src/settings_gui.cpp

```diff
@@ -583,16 +707,22 @@ struct GameOptionsWindow : Window {
 
 	void SetTab(WidgetID widget)
 	{
-		this->SetWidgetsLoweredState(false, WID_GO_TAB_GENERAL, WID_GO_TAB_GRAPHICS, WID_GO_TAB_SOUND, WID_GO_TAB_SOCIAL);
+		this->SetWidgetsLoweredState(false, WID_GO_TAB_GENERAL, WID_GO_TAB_GRAPHICS, WID_GO_TAB_FONTS, WID_GO_TAB_SOUND, WID_GO_TAB_SOCIAL);
 		this->LowerWidget(widget);
 		GameOptionsWindow::active_tab = widget;
 
 		int pane;
 		switch (widget) {
 			case WID_GO_TAB_GENERAL: pane = 0; break;
 			case WID_GO_TAB_GRAPHICS: pane = 1; break;
+#ifdef HAS_TRUETYPE_FONT
+			case WID_GO_TAB_FONTS: pane = 2; break;
+			case WID_GO_TAB_SOUND: pane = 3; break;
+			case WID_GO_TAB_SOCIAL: pane = 4; break;
+#else
 			case WID_GO_TAB_SOUND: pane = 2; break;
 			case WID_GO_TAB_SOCIAL: pane = 3; break;
+#endif
```

Comment:
You don't hide any widgets when HAS_TRUETYPE_FONT is not set, so this is wrong.

The simplest approach would be to just hide WID_GO_TAB_FONTS and leave the rest alone.

Reply:
I messed up.  Anyways I've tracked down all of the uses of the font tab and guarded them as well.  I had to hack a desktop build disabling Freetype font support but I tested it and everything seems fine now.  I'll check the preview build when it's done though.

Reply:
The tab isn't visible on the preview build and the other tabs still work fine. Seems weird but as far as I know the preview build builds without freetype font support so this be weird but "works as intended".